e2e/e2eutils/secrets.go

```diff
@@ -15,17 +15,17 @@ import (
 // We prefer a mnemonic rather than direct private keys to make it easier
 // to export all testing keys in external tooling for use during debugging.
 var DefaultMnemonicConfig = &MnemonicConfig{
-	Mnemonic:     "test test test test test test test test test test test junk",
-	Deployer:     "m/44'/60'/0'/0/1",
-	CliqueSigner: "m/44'/60'/0'/0/2",
-	Validator:    "m/44'/60'/0'/0/3",
-	Challenger:   "m/44'/60'/0'/0/4",
-	Batcher:      "m/44'/60'/0'/0/5",
-	ProposerP2P:  "m/44'/60'/0'/0/6",
-	Alice:        "m/44'/60'/0'/0/7",
-	Bob:          "m/44'/60'/0'/0/8",
-	Mallory:      "m/44'/60'/0'/0/9",
-	SysCfgOwner:  "m/44'/60'/0'/0/10",
+	Mnemonic:         "test test test test test test test test test test test junk",
+	Deployer:         "m/44'/60'/0'/0/1",
+	CliqueSigner:     "m/44'/60'/0'/0/2",
```

Comment:
By the way, do we still need this one?

Reply:
The `CliqueSigner` is required by the L1 geth when running e2e test.